pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -1055,13 +1055,13 @@ def __gt__(self, other):
 
         t = Timedelta("1s")
 
-        assert not (t == "string")
-        assert not (t == 1)
-        assert not (t == CustomClass())
-        assert not (t == CustomClass(cmp_result=False))
+        assert t != "string"
+        assert t != 1
+        assert t != CustomClass()
+        assert t != CustomClass(cmp_result=False)
 
         assert t < CustomClass(cmp_result=True)
-        assert not (t < CustomClass(cmp_result=False))
+        assert t >= CustomClass(cmp_result=False)
```

Comment:
If you look at test failures, looks like this refactor is a problem. `CustomClass` only defines greater than, so best to just leave as is

Reply:
would
```
        assert not (CustomClass(cmp_result=False) > t)
```
work?

Reply:
Okay. It seems like there are some less than operators that are fine too. Any preference between the original
```
assert not t < CustomClass(cmp_result=False)
```
or
```
assert not CustomClass(cmp_result=False) > t
```
?

Reply:
No strong preference but probably lean toward the first one as clearer to the test intent

Reply:
Agreed. Thanks.